interactions/client/bot.py

```diff
@@ -1092,6 +1093,26 @@ def reload(
         self.remove(name, package)
         return self.load(name, package, *args, **kwargs)
 
+    async def modify(
+        self,
+        username: Optional[str] = MISSING,
+        avatar: Optional[Image] = MISSING,
+    ) -> User:
+        """
+        Modify the bot user account settings.
+
+        :param username?: The new username of the bot
+        :type username?: Optional[str]
+        :param avatar?: The new avatar of the bot
+        :type avatar?: Optional[Image]
+        :return: The modified User object
+        :rtype: User
+        """
+        payload: dict = {"username": username, "avatar": avatar.data}
+        data: dict = await self._http.modify_self(payload=payload)
+
+        return User(**data)
+
```

Comment:
I think this should be moved underneath `get_extension()` because this function is basically in between two functions related to `Extension`s.

Reply:
I can move it